navigation/navigation-runtime/src/webMain/kotlin/androidx/navigation/BrowserHistory.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.navigation
+
+import androidx.core.bundle.Bundle
+import androidx.navigation.NavBackStackEntry
+import androidx.navigation.NavController
+import androidx.navigation.NavGraph
+
+expect abstract class BrowserWindow
+
+/**
+ * Configures the browser navigation for the given window and navigation controller.
+ *
+ * @param window an instance of browser's window to be configured
+ * @param navController an instance of NavController handling the navigation logic
+ */
+expect fun configureBrowserNavigation(window: BrowserWindow, navController: NavController)
```

Comment:
Can we have some tests for this funciton/logic?

Reply:
done